src/ui/TreeView.js

```diff
@@ -50,8 +50,8 @@ class TreeView {
     return yo`<ul style=${this.cssList}>${children}</ul>`
   }
 
-  formatDataDefault (key, self, children, expand) {
-    var label = yo`<span style=${this.cssLabel}><label style='width: 10px'></label><label>${key}: ${self}</label></span>`
+  formatDataInternal (key, data, children, expand) {
```

Comment:
this is not obvious in the diff, but `formatDataDefault` is still present.
here, `formatDataInternal` is only called by the class but `formatData` is the function that should be overriden (by default it is `formatDataDefault`)